addons/ui/fnc_initDisplayRemoteMissions.sqf

```diff
@@ -89,8 +89,10 @@ private _fnc_storeMapMissions = {_this spawn {isNil { // delay a frame
     private _ctrlMissions = _display displayCtrl IDC_SERVER_MISSION;
 
     private _missions = [];
+    private _fnc_replace = uiNamespace getVariable "CBA_fnc_replace";
+
     for "_i" from 0 to (lbSize _ctrlMissions - 1) do {
-        private _name = _ctrlMissions lbText _i;
+        private _name = [_ctrlMissions lbText _i, "%20", " "] call _fnc_replace; // replace %20 with space
```

Comment:
looks like `_fnc_replace` is bad name

Reply:
How so? It is `CBA_fnc_replace` fetched from ui namespace in case the host started the game with -world=empty.

Reply:
oh sorry I just missed assigning line :confused: :man_facepalming: